projects/observability/src/shared/components/cartesian/cartesian-chart.component.scss

```diff
@@ -104,6 +104,10 @@
     }
 
     .legend-entries {
+      /* Prevent large legends from shrinking chart */
+      max-height: 3.5rem;
```

Comment:
Two things:
1. we do not use the relative size units anywhere (`em` is used in one place)
2. Since legend text and other things (children of this class)  are fixed (`px`) width and height, here it will not give us the benefit of using the relative size.

Maybe we can use a fixed height here in `px`, you can try and convert this `3.5rem` into `px` by looking at the dom.

Reply:
Sure. One question, should the height be assigned to fit 2 rows exactly (screenshot in PR description), or show a little bit of the third row, like this?

<img width="961" alt="scroll" src="https://user-images.githubusercontent.com/29686866/200995081-13de595f-0e7d-4985-975a-b4c7aa80185c.png">